system/environment variables getter with fallback/2ac2a613-e930-4e37-9a28-02ce0421836b.json

```diff
@@ -0,0 +1,186 @@
+{
+  "name": "system/Environment variables getter with fallback",
+  "interface": {
+    "inputs": {
+      "a51796cd-95b6-43e6-88e8-54ae80e59646": {
+        "type": "string[]",
+        "name": "variable names",
+        "description": "Receives a list of variable names to be resolved from the environment.\n\nExample:\n[\"Var1\", \"Var2\"]"
```

Comment:
Please add tag to the example, like described here: https://kwaia.atlassian.net/wiki/spaces/CRANQ/pages/804356131/CRANQ+Component+Documentation#Examples
Something like: 
`["Var1", "Var2"]@0`